Tests/run.sh

```diff
@@ -25,7 +25,7 @@ else
 	exit 1
 fi
 
-
+sleep 10
```

Comment:
it's because of this https://github.com/keboola/connection/issues/1925 the connection needs some time to realize that the files were deleted
all this is completely removed in the next PR

Reply:
ah, gotcha, thanks